.github/PULL_REQUEST_GUIDELINES/rule_tuning_guidelines.md

```diff
@@ -0,0 +1,22 @@
+## Rule: Tuning - Guidelines
+
+These guidelines serve as a reminder set of considerations when tuning an existing rule.
+
+### Documentation and Context
+
+- [ ] Detailed description of the suggested changes.
+- [ ] Provide example JSON data or screenshots.
+- [ ] Provide evidence of FP (False Positive) reduction.
+- [ ] Ensure the tuning includes necessary updates to the release documentation and versioning.
+
+### Rule Metadata Checks
+
+- [ ] `updated_date` matches the date of tuning PR merged.
+- [ ] `min_stack_version` should support the widest stack versions.
+- [ ] `name` and `description` should be descriptive and not include typos.
+- [ ] `query` should be inclusive, not overly exclusive.
```

Comment:
I'd add note here as well. Sometimes when tuning, the original intent of the rule may or may not change and thus we have to review the notes to ensure these still align.